### PR TITLE
Adds beam example to readthedocs and Makefile

### PR DESCRIPTION
**Pull request by pgkirsch**

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -48,6 +48,22 @@ The output is
...
+The output is
+
+.. literalinclude:: examples/beam_output.txt
+
+By plotting the deflection, we can see that the agreement between the analytical solution and the GP solution is good.
+
+.. figure:: examples/beam.png
+   :width: 500 px
+   :align: center
+
```
Comment (whoburg) [inline docs/source/examples.rst]: please turn the figure into a vector graphics format, either svg or pdf. Often you can print directly to one of those, or you can print to eps and then run 

epstopdf beam.eps

at the command line.

File: docs/source/examples/beam_output.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ w : [  -        0.0173    0.0653    0.137    ... ]  [m]   Displacement   
+
+Constants
+---------
+EI : 1e+04                                           [N*m**2]                
+dx : 0.5556                                          [m]                     
+ M : [  -         -         -         -       ... ]  [N*m]    Internal moment
+ V : [  -         -         -         -       ... ]  [N]      Internal shear 
+th : [ 1e-16      -         -         -       ... ]           Slope          
+ w : [ 1e-16      -         -         -       ... ]  [m]      Displacement
```
Comment (whoburg) [inline docs/source/examples/beam_output.txt]: hmm, substituting one element of a vector variable sure makes for overly-complicated output. Not your fault though!

**Comment by whoburg** [comment]
code review complete -- only action item is changing the fig to vector graphics.

**Comment by pgkirsch** [comment]
Ah whoops. Fixed.

**Comment by whoburg** [comment]
looks great!

**Pull request merged**